### PR TITLE
Updated URL data for my mods.

### PR DESCRIPTION
**Pull request by Arthmoor**
Hopefully I've done this right. :P

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -14078,7 +14085,10 @@ plugins:
         util: 'SSEEdit v4.0.3'
 
   - name: 'Skyshards.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/6331/' ]
+    req: [ 'Unofficial Skyrim Special Edition Patch.esp' ]
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/1939-eso-skyshards/'
+	  - 'https://bethesda.net/en/mods/skyrim/mod-detail/3326552/'
```
Comment (pStyl3) [inline masterlist.yaml]: This url needs to be indented the same length than the url above, so two empty spaces compared to `url` are needed.

**Comment by pStyl3** [comment]
Thanks!

**Comment by sibir-ine** [comment]
@Arthmoor Just FYI, if a mod has another mod as a master, you don't need to add a requirement message for it; LOOT will display a missing masters message automatically. I also noticed that **Sounds of Skyrim - Complete Edition.esp** already has **C.Acoustic** in its description.

**Comment by Arthmoor** [comment]
Ah, ok. I didn't know the information wasn't needed so figured I'd add it while I was at it.

**Pull request merged**